source/templates/handlebars-basics.md

```diff
@@ -1,64 +1,72 @@
 Ember.js uses the [Handlebars templating library](http://www.handlebarsjs.com)
-to power your app's user interface. Handlebars templates are just like
-regular HTML, but also give you the ability to embed expressions that
-change what is displayed.
+to power your app's user interface. Handlebars templates contain static HTML and
+dynamic content inside what we call mustaches: `{{}}`.
```

Comment:
"double curly braces" does not seem an optimal replacement for all uses in this document. Having a term is useful.

I really dislike "Handlebars expression", though that is what http://handlebarsjs.com/ introduces them as. In conversation we never use that term though, and mustache is a lot more common. I'd hoped to start unifying that common parlance with the language we use in the guides. Historically even Handlebars ties back to Mustache.js, from whence the name arises.

Regardless I've fallen back to "Handlebars expression", and audited the PR to remove usages of mustache.